examples/DesktopDisplay_SonicDisc/README.md

```diff
@@ -0,0 +1,6 @@
+
+
+# DesktopDisplay_SonicDisc
```

Comment:
To someone (like me) that has not used Processing before, is it straight forward how to run this? Are there any necessary dependencies or would I just need to download Processing open the project and click run?

Reply:
good point, I've added a link to the processing webpage to download it

Reply:
Sounds exciting, but where is the commit? :smiley: 

Looking forward to merge this!